decision_service/unit_test/ds_eventhub_client_test.cc

```diff
@@ -0,0 +1,36 @@
+#define BOOST_TEST_DYN_LINK
+#ifdef STAND_ALONE
+#   define BOOST_TEST_MODULE Main
+#endif
+
+#include "http_server/stdafx.h"
+#include "logger/ds_eventhub_client.h"
+#include "http_server/http_server.h"
+
+#include <boost/test/unit_test.hpp>
+
+using namespace web;
+using namespace http;
+using namespace utility;
+using namespace http::experimental::listener;
+
+
+using namespace decision_service;
+
+
+BOOST_AUTO_TEST_CASE(send_something)
+{
+	//start a http server that will receive events sent from the eventhub_client
+	http_helper http_server;
+	http_server.on_initialize(U("http://localhost:8080"));
```

Comment:
>U [](start = 27, length = 1)

isn't that a macro provided by cpprest? do we want to leak that?

Reply:
i think this macro is provided by the standard library, not cpprest:
http://en.cppreference.com/w/cpp/language/string_literal